semcore/d3-chart/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [3.23.1] - 2024-01-16
+
+### Fixed
+
+- method for find next focusable element after chart plot.
```

Comment:
Method that finds next focusable element is an internal tool that assists the user focus flow. Basically you need to write what bugs are fixes

Reply:
fixed